python/cudf/cudf/core/series.py

```diff
@@ -5266,3 +5266,15 @@ def isclose(a, b, rtol=1e-05, atol=1e-08, equal_nan=False):
         result_col[equal_nulls] = True
 
     return Series(result_col, index=index)
+
+
+def _get_unique(df, dummy_na, name):
+    if isinstance(df[name]._column, cudf.core.column.CategoricalColumn):
+        unique = df[name]._column.categories
```

Comment:
Can you add a docstring to this function? A categorical column doesn't need to necessarily use every value in its categories, is that an issue?

Reply:
Remove this instance of function to de-duplicate. 

Regarding the categorical column, yes we would need to return the categories instead of just the existing unique values, for example:
```python
>>> import pandas as pd
>>> pd.CategoricalDtype(['a','b','c'])
CategoricalDtype(categories=['a', 'b', 'c'], ordered=False)
>>> dtype = pd.CategoricalDtype(['a','b','c'])
>>> s = pd.Series([], dtype=dtype)
>>> s
Series([], dtype: category
Categories (3, object): ['a', 'b', 'c'])
>>> pd.get_dummies(s)
Empty DataFrame
Columns: [a, b, c]
Index: []
>>> s.unique()
[], Categories (0, object): []
```